api/auth_token.go

```diff
@@ -77,10 +86,11 @@ func (c *TokenAuth) LookupSelf() (*Secret, error) {
 }
 
 func (c *TokenAuth) Renew(token string, increment int) (*Secret, error) {
-	r := c.c.NewRequest("PUT", "/v1/auth/token/renew/"+token)
-
-	body := map[string]interface{}{"increment": increment}
-	if err := r.SetJSONBody(body); err != nil {
+	r := c.c.NewRequest("PUT", "/v1/auth/token/renew")
```

Comment:
`PUT` is actually more widely used. However, I actually like having both in the API, as it serves as an early smoke test if we accidentally break PUT/POST compatibility.